tests/pipeline/test_pipeline_load_info.py

```diff
@@ -0,0 +1,26 @@
+import dlt
+
+
+data = [
+    {"id": 1, "name": "Alice"},
+    {"id": 2, "name": "Bob"},
+]
+
+
+def test_pipeline_load_info_metrics_schema_is_not_chaning() -> None:
+    pipeline = dlt.pipeline(
+        pipeline_name="quick_start",
+        destination="duckdb",
+        dataset_name="mydata",
+    )
+
+    load_info = pipeline.run(data, table_name="users")
```

Comment:
please load something more complicated than this. ie. source with a resource that have several hints